webnode/config/webpack.config.dev.js

```diff
@@ -1,96 +1,29 @@
 "use strict";
```

Comment:
A lot of the comments removed from this file seem like they might have been helpful?

Reply:
they are helpful when you are porting over, if you dont know much about webpack. If you know webpack its similar to writing a comment that you are about to loop over an object.